src/scripts/clipperUI/componentBase.ts

```diff
@@ -132,10 +126,36 @@ export abstract class ComponentBase<TState, TProps> {
 				element.style.outlineStyle = "none";
 			},
 			tabIndex: tabIndex,
-			"data-setnameforarrowkeynav": setNameForArrowKeyNav
+			"data-setnameforarrowkeynav": setNameForArrowKeyNav,
+			"data-defaultaction": defaultAction
```

Comment:
If setNameForArrowKeyNav/defaultAction is undefined, do the attributes still get set, but with undefined values? We should make sure these attributes do not get set if they are undefined.